NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java

```diff
@@ -309,4 +309,12 @@ public static void hideProjectilesFromHiddenPlayers() {
         hideProjectilesFromHiddenPlayers = getBoolean("settings.hide-projectiles-from-hidden-players", false);
         c.addComment("settings.hide-projectiles-from-hidden-players", "Hides projectiles from hidden players");
     }
+
+    public static boolean antiEnderPearlGlitch;
+    
+    public static void antiEnderPearlGlitch() {
```

Comment:
The method always has to be private static or else it won't be detected. Also make sure to update the configuration version when you make a new config.